src/i18n/definitions.js

```diff
@@ -491,15 +493,18 @@ let strings = new LocalizedStrings(
       wordSourceDefaultText: "din tidligere læsning",
       wordSourcePrefix: "Ord fra",
 
-      //FindWordsInContext
+      //ExerciseHeadlines
       findTheWordInContextHeadline: "Find ordet/ordene i sammenhængen",
+      chooseTheWordFittingContextHeadline:
+        "Vælg ordet/ordene, der passer i sammenhængen",
 
       //FeedbackButtons
       giveFeedback: "Feedback på øvelsen",
```

Comment:
I would say: "Giv feedback" eller "Giv feedback på øvelsen". Without the verb "Giv" you are missing the call for action.